templates/llm-router/README.ipynb

```diff
@@ -0,0 +1,1038 @@
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>169249</th>\n",
+       "      <td>Write a comparison essay of at least 500 words...</td>\n",
+       "      <td>[ultrachat]</td>\n",
+       "      <td>Title: A Comparative Analysis of Driving a Car...</td>\n",
+       "      <td>Title: The Great Debate: Driving a Car vs. Ri...</td>\n",
+       "      <td>3</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>116934</th>\n",
+       "      <td>Q: You are provided with an \"Event\", \"Intent\" ...</td>\n",
+       "      <td>[flan_v2_niv2]</td>\n",
+       "      <td>PersonX might feel satisfied or content using ...</td>\n",
+       "      <td>Person X probably feels comfortable and satis...</td>\n",
+       "      <td>5</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                                                   prompt           source  \\\n",
+       "6062    Based on the features mentioned, which hotel d...  [evol_instruct]   \n",
+       "113830  Provide step-by-step instructions on how to cr...      [ultrachat]   \n",
+       "138869  What are the 10 largest cities in the US by po...  [lmsys-chat-1m]   \n",
+       "169249  Write a comparison essay of at least 500 words...      [ultrachat]   \n",
+       "116934  Q: You are provided with an \"Event\", \"Intent\" ...   [flan_v2_niv2]   \n",
+       "\n",
+       "                                                     gpt4  \\\n",
+       "6062    Based on the features mentioned, Hotel A seems...   \n",
+       "113830  Sure, here's a simple step-by-step guide on ho...   \n",
+       "138869  As of the most recent data available, the 10 l...   \n",
+       "169249  Title: A Comparative Analysis of Driving a Car...   \n",
+       "116934  PersonX might feel satisfied or content using ...   \n",
+       "\n",
+       "                                                  mixtral  label  \n",
+       "6062     Based on the information provided, I would sa...      5  \n",
+       "113830   Sure, I'd be happy to help you make a homemad...      4  \n",
+       "138869   Sure, I'd be happy to help with that! As of t...      5  \n",
+       "169249   Title: The Great Debate: Driving a Car vs. Ri...      3  \n",
+       "116934   Person X probably feels comfortable and satis...      5  "
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from online_inference import parse_judge_responses\n",
+    "\n",
+    "labels, explanations = parse_judge_responses(judge_responses)\n",
+    "dataset_df[\"label\"] = dataset_df.index.map(labels)\n",
+    "dataset_df.to_json(\"assets/labeled_test_dataset.json\")\n",
+    "dataset_df.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Assume we labeled the full dataset (~120K samples)\n",
```

Comment:
Some where mention that the notebook is running this data collection at small scale. 

For larger scale you need to do x y z, also this is how much it'll cost, etc.